jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/comments/CaseCommentsViewImpl.html

```diff
@@ -52,7 +52,10 @@ <h2 class="card-pf-title">
                         </div>
                     </div>
                 </div>
-                <div data-field="pagination"/>
+                <div id="load-more" data-field="load-div" style="text-align: center; padding-top: 10px;">
+                    <button type="button" class="btn btn-primary" data-field="load-more-comments">Load More Comments</button>
```

Comment:
As general good practice, I would recommend removing the expected i18n String (Load More Comments) so we can ensure that the key is pickup by the translation service.